config/300-filter-crd.yaml

```diff
@@ -4,7 +4,7 @@
 # you may not use this file except in compliance with the License.
 # You may obtain a copy of the License at
 #
-#     https://www.apache.org/licenses/LICENSE-2.0
+#     http://www.apache.org/licenses/LICENSE-2.0
```

Comment:
Does it matter if using http vs. https, or did the linter complain?

Reply:
It complained yes. The only thing it ignores is the year.

I don't think it matters personally, but I won't why these 4 specific files had HTTPS.

Reply:
Guess it depends on what was used as the basis for the copy/pasta boilerplate.